internal/namespaces/init/init.go

```diff
@@ -231,6 +233,24 @@ func initCommand() *core.Command {
 				args.InstallAutocomplete = scw.BoolPtr(installAutocomplete)
 			}
 
+			// Ask whether to remove v1 configuration file is it exists
```

Comment:
I would move this whole portion to the end of the run

Reply:
It is not consistent with other flags